bundesliga_app/views.py

```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+from django.views.generic.base import TemplateView
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.views.generic.list import ListView
```

Comment:
Is a good practice to order imports alphabetically.

Reply:
going to be fixed! thanks!